tests/smoke/desiredstate_test.go

```diff
@@ -65,9 +65,12 @@ func TestDesiredState(t *testing.T) {
 	output, stderr, err := test.RunPorter("installation", "apply", "mybuns.yaml", "--namespace", "operator")
 	require.NoError(t, err)
 	require.Contains(t, stderr, "The installation is out-of-sync, running the install action")
-	require.Contains(t, stderr, "Triggering because the installation has not completed successfully yet")
-	test.RequireInstallationExists("operator", "mybuns")
+	require.Contains(t, stderr, "Triggering because the installation has not completed successfully yet", stderr)
```

Comment:
Did you intend to have stderr  added as the last argument?

Reply:
Oops, I removed it